### PR TITLE
add package.json

### PR DESCRIPTION
**Pull request by mathisonian**
Moving this issue to dev branch re #5492 

> This will allow browserify users to install three.js directly using `npm`.
> 
> The package can be installed straight from github by running `npm install mrdoob/three.js` (once this pull request is merged. for now `npm install mathisonian/three.js` works).
> 
> It would also be cool to publish the package to npm. It looks like someone [already published a browserify compatible three.js 2 years ago](https://www.npmjs.org/package/three.js), maybe we can get they will give up ownership of that package name to make space for the official one.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "three.js",
+  "version": "0.68.0",
```
Comment (mrdoob) [inline package.json]: Shouldn't this be `"0.70.0"`?

**Comment by mrdoob** [comment]
Hmmm, we have the file [bower.json](https://github.com/mrdoob/three.js/blob/master/bower.json) which has a very similar json structure. Isn't it possible to have just one file?

**Comment by mathisonian** [comment]
Unfortunately not really... see [d3](https://github.com/mbostock/d3) for an example of how they are supporting multiple js package managers. Browserify won't know about `bower.json`, and I don't think bower looks for `package.json` files

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "three.js",
+    "3d",
+    "webgl"
+  ],
+  "author": "mrdoob",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mrdoob/three.js/issues"
+  },
+  "homepage": "https://github.com/mrdoob/three.js"
```
Comment (mrdoob) [inline package.json]: I think this should be http://threejs.org?

**Comment by bripkens** [comment]
+1: This would be really nice to have for our current project.

**Comment by sithCC** [comment]
agree to bripkens! with this feature we can handle our projects more comfortable.

**Comment by mcellier** [comment]
+1: currently, we're using napa (https://www.npmjs.com/package/napa) to install packages without package.json... It's kind of ugly.

**Comment by bripkens** [comment]
Same here, also, there is no need to download all examples when you are only interested in the main library.

**Comment by mcellier** [comment]
Yes, but some renderers (CanvasRenderer & CSS3DRenderer) are only included in examples

**Pull request closed without merging**